rust/main/agents/scraper/migration/src/m20230309_000001_create_table_domain.rs

```diff
@@ -462,6 +462,22 @@ const DOMAINS: &[RawDomain] = &[
         is_test_net: true,
         is_deprecated: false,
     },
```

Comment:
Something I'm wondering - is the scraper indexing the SVM IGP payments and deliveries in a sequence aware fashion, or using the rate limited / watermark approach?

In the relayer it indexes SVM IGP payments in a sequence aware manner, but other ones in a rate limited / watermark approach. We'll want to do the same in the scraper otherwise I'd expect bugs to occur

For reference, this is how we do it in the relayer:

We create ContractSyncs here https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/5db46bd315de02f1c7514aeee24aec349b6b8f80/rust/main/agents/relayer/src/relayer.rs#L161, which calls https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/5db46bd315de02f1c7514aeee24aec349b6b8f80/rust/main/hyperlane-base/src/settings/base.rs#L205-L250 that creates a `sequenced_contract_sync` if the CursorType is SequenceAware (the case for SVM IGP payments), and a `watermark_contract_sync` otherwise

The db used by the sequence aware cursors also requires it to impl `HyperlaneSequenceAwareIndexerStoreReader` https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/5db46bd315de02f1c7514aeee24aec349b6b8f80/rust/main/hyperlane-base/src/contract_sync/cursors/sequence_aware/forward.rs#L31 which iiuc is only implemented for the Scraper's SQL db for messages atm https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/5db46bd315de02f1c7514aeee24aec349b6b8f80/rust/main/agents/scraper/src/chain_scraper/mod.rs#L393